awx/ui/test/e2e/tests/test-workflow-visualizer.js

```diff
@@ -5,39 +5,43 @@ import {
     getWorkflowTemplate
 } from '../fixtures';
 
-let data;
-const spinny = "//*[contains(@class, 'spinny')]";
-const workflowTemplateNavTab = "//at-side-nav-item[contains(@name, 'TEMPLATES')]";
-const workflowSelector = "//a[contains(text(), 'test-actions-workflow-template')]";
-const workflowVisualizerBtn = "//button[contains(@id, 'workflow_job_template_workflow_visualizer_btn')]";
-const workflowSearchBar = "//input[contains(@class, 'SmartSearch-input')]";
+const spinny = 'div.spinny';
+const workflowTemplateNavTab = 'at-side-nav-item i[class*="pencil"]';
+const workflowTemplateVisualizerBtn = '#workflow_job_template_workflow_visualizer_btn';
+const workflowSearchBar = 'div[ui-view="templatesList"] smart-search input';
 const workflowText = 'name.iexact:"test-actions-workflow-template"';
-const workflowSearchBadgeCount = '//span[contains(@class, "at-Panel-headingTitleBadge") and contains(text(), "1")]';
```

Comment:
note: I added a global `search` function to `e2e/commands` so I converted most of these to css